src/services/ui/src/pages/actions/template.tsx

```diff
@@ -81,6 +81,21 @@ export const ActionFormTemplate = <D extends FieldValues>({
             required: requireAddlInfo,
           })}
         />
+        {Object.keys(formController.formState.errors).length !== 0 && (
+          <Alert className="my-6" variant="destructive">
+            Input validation error(s)
+            <ul className="list-disc">
+              {Object.values(formController.formState.errors).map(
+                (v, idx) =>
```

Comment:
Could we maybe change v to something like `error` or `err` so down the road when I look back at this it resonates with me more?

Reply:
Yeah, I'll tack this on with the variable name change in `superRefine` from `val` -> `data`